### ISSUE TITLE
Temporary content directory god mode

### ISSUE THREAD

**Issue by bedeho** [Joystream/joystream#2577]
Both for populating the content after redeploying the storage system, such as in `Giza` network launch, but also to support planned marketing initiatives where we can have auto-synching with YouTube for creator with a pre-existing presence, we need a way to use SUDO to

- create a channel for a member
- update a channel
- publish a video in a channel
- update a video
- delete a video

This is functionality which is temporary before mainnet, so we can just introduce ability to use SUDO for these extrinsics

**Comment by bedeho** [comment]
I think there may be a more useful way to solve this which also aligns with some long features the creators likely will have.

Allow a channel owner to have up to a fixed number of other members as _collaborators_ on a channel, meaning that they are allowed to do anything the owner can do. In the future we could add permissions, but lets start simple. This is a very realistic feature, as people who own channels may want to delegate part of the work.

With this feature, we can do the bootstrapping and marketing stuff we need by just having a designated `Jsgenesis` member setup as a collaborator during channel creation. There is no need for a SUDO privilege. Even the channel creation, which is done by a member (not a channel owner or collaborator, like the others) does not require SUDO, we can just create the member at our own expense with our account, with the collaborator set, and then we change the controller account of the member to the provided account from the new user.

**Comment by dobertRowneySr** [comment]
The collaborator set is a `BTreeSet<MemberId>`, that can be created and update together with the channel by providing a the set during creation/update (so that is responsibility of the caller to ensure that the set is valid).
This scheme is the one I used on the previous PRs.
As far as overlapping between channel owner and collaborator goes I have found that only video operation {update, create, delete} needs collaboration so far. Since:
- playlist extrinsics are yet to be implemented 
- same thing with `-person` extrinsics
- members are restricted from censoring and managing categories
- feature videos can only be set by Lead

I had to split up the `actor_authorized_to_update_channel` into subchecks in order to reuse them
